src/helpers/image-helper.php

```diff
@@ -46,6 +50,41 @@ class Image_Helper {
 	 */
 	private $url_helper;
 
+	/**
+	 * Holds the home_url() value to speed up loops.
+	 *
+	 * @var string
+	 */
+	protected $home_url = '';
```

Comment:
It's weird that this should be in this class. Shouldn't this be taken from elsewhere, and maybe made public there?